flags.go

```diff
@@ -1,21 +1,17 @@
 package main
 
 import (
-	"os"
-	"os/user"
-	"path"
-
 	"github.com/codegangsta/cli"
-
-	log "github.com/Sirupsen/logrus"
+	"os"
+	"path/filepath"
+	"runtime"
 )
 
 func homepath(p string) string {
-	usr, err := user.Current()
-	if err != nil {
-		log.Fatal(err)
+	if runtime.GOOS == "windows" {
```

Comment:
nitpick:

Instead of repeating that `filepath.Join` twice, how about something like this?

```go
var home string
switch runtime.GOOS {
default:
    home = os.Getenv("HOME")
case "windows":
    home = os.Getenv("USERPROFILE")
}
return filepath.Join(home, p)
```